css/partners.css

```diff
@@ -0,0 +1,164 @@
+#partners {
```

Comment:
I have a feeling that it makes sense to make css classes instead of ids out of the most definitions you pated from here to line 84.